### PR TITLE
ruv.json : Icelandic broadcasting service

### PR DESCRIPTION
**Pull request by karlo2105**
RUV is public broadcasting service from Iceland. It consists of two channels : RUV and RUV 2.

### CODE REVIEW

File: datafiles/streams/ruv.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "geolocked": true,
+    "region": "IS",
+    "language": "isl",
+    "hd": true
+  },
+  "streams": [
+    {"name": "ruv",
+      "title": "RUV",
+      "url": "http://ruv.is/ruv"},
+    {"name": "lasexta",
```
Comment (beardypig) [inline datafiles/streams/ruv.json]: Should be RUV2? I can update it if you like.

Comment (karlo2105) [reply datafiles/streams/ruv.json]: Yes of course. ;)

Comment (beardypig) [reply datafiles/streams/ruv.json]: Want me to change it, or do you want to do it?

**Comment by beardypig** [comment]
Cool, thanks @karlo2105 :D

**Pull request merged**